bin/bump-version

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+usage() { echo "Usage: $0 -p [major | minor | patch]" 1>&2; exit 1; }
+
+while getopts "p:" o; do
+  case "${o}" in
+    p)
+      patch_level=${OPTARG}
+      (( patch_level == 'major' || patch_level == 'minor' || patch_level == 'patch'))
+      ;;
+    *)
+      usage
+      ;;
+  esac
+done
+
+echo "$patch_level"
+
+if [[ -z "${patch_level}" ]]; then
+  usage
+fi
+
+new_version=$(npm version "${patch_level}" --no-git-tag-version)
+git checkout -b "${new_version}"-release-notes
+git add package.json package-lock.json
+git commit -m "${new_version}"
```

Comment:
Yeah, that's a good point. My reasoning for going ahead with the commit here is that we're essentially just augmenting `npm version patch` which goes the commit _and adds a tag_, where we'd prefer a release branch for review first.

Since the package changes are explicitly checked in without anything else added, this feels like a safe assumption where little can go wrong. In dependabot-core, we hold the actual checkin as there's always the chance the Changelog script we wrote ourselves misfires for example 🤔